app/components/md-editor.js

```diff
@@ -1,17 +1,13 @@
-import Component from '@ember/component';
+import Component from '@glimmer/component';
 import { action } from '@ember/object';
-import { tracked } from '@glimmer/tracking';
 
 export default class MdEditorComponent extends Component {
   classNames = ['md-editor-and-toolbar'];
```

Comment:
This does not seem to work properly. The class is not added, causing there to be no line around the element.
Current:
![image](https://user-images.githubusercontent.com/7385023/204094491-da5793a9-248e-461f-9305-9f81e106ab3c.png)
New:
![image](https://user-images.githubusercontent.com/7385023/204094478-d67f8f42-3b33-44c6-b45f-ed766914809f.png)